src/Illuminate/Console/LaravelStyle.php

```diff
@@ -0,0 +1,165 @@
+
+		parent::__construct($input, $output);
+	}
+
+	/**
+	 * Formats informational text.
+	 *
+	 * @param string|array $messages
+	 */
+	public function text($messages)
+	{
+		foreach ((array) $messages as $message)
+		{
+			$this->writeln($message);
+		}
+	}
+
+	/**
+	 * Formats a success result bar.
+	 *
+	 * @param string|array $message
+	 */
+	public function success($message)
+	{
+		$this->styledText($message, 'info');
+	}
+
+	/**
+	 * Formats a note admonition.
+	 *
+	 * @param string|array $message
+	 */
+	public function note($message)
+	{
+		$this->styledText($message, 'comment');
+	}
+
+	/**
+	 * Formats an error result bar.
+	 *
+	 * @param string|array $message
+	 */
+	public function error($message)
+	{
+		$this->styledText($message, 'error');
+	}
+
+	/**
+	 * Formats an warning result bar.
+	 *
+	 * @param string|array $message
+	 */
+	public function warning($message)
+	{
+		$this->borderedBlock($message, 'comment');
+	}
+
+	/**
+	 * Formats a caution admonition.
+	 *
+	 * @param string|array $message
+	 */
+	public function caution($message)
+	{
+		$this->borderedBlock($message, 'error', '!');
+	}
+
+	/**
+	 * Formats text with a given style.
+	 *
+	 * @param string|array $messages
+	 * @param string|null $style
+	 */
+	protected function styledText($messages, $style = null)
+	{
+		foreach ((array) $messages as $message)
+		{
+			if ($style)
+			{
+				$message = sprintf('<%s>%s</>', $style, $message);
+			}
+
+			$this->writeln($message);
+		}
+	}
+
+	/**
+	 * Formats a border block with a given style.
+	 *
+	 * @param string|array $messages
+	 * @param string|null $style
+	 * @param string $borderChar
+	 * @param int $paddingSize
+	 */
+	protected function borderedBlock($messages, $style = null, $borderChar = '*', $paddingSize = 3)
+	{
+		$messages = (array) $messages;
+		$lines = array();
+		$prefix = $borderChar . str_repeat(' ', $paddingSize);
+
+		// Get the length of the longest string
+		$length = max(array_map('Symfony\Component\Console\Helper\Helper::strlen',  $messages));
+		$length = min($length, $this->lineLength - 2 * $paddingSize - 2);
+
+		foreach ($messages as $key => $message)
+		{
+			// Explode the message if it's too long.
+			$message = OutputFormatter::escape($message);
+			$parts = explode("\n", wordwrap($message, $length));
+
+			// Add new line after each message, if not only 1 message
+			if (count($messages) > 1 && $key < count($messages) - 1) {
+				$parts[] = '';
+			}
+
+			// Add the parts with a prefix/suffix
+			foreach ($parts as $part)
+			{
+				$suffix = str_repeat(' ', $paddingSize + $length - Helper::strlen($part)) . $borderChar;
+				$lines[] = $prefix . $part . $suffix;
+			}
+		}
+
+		// Add border as first and last item in the array
+		$border = str_repeat($borderChar, $length + 2 * $paddingSize + 2);
+		$lines[] = $border;
+		array_unshift($lines, $border);
+
+		// Write all the messages in the correct style
+		$this->styledText($lines, $style);
+	}
+
+	/**
```

Comment:
Add the descriptions, void returns, double phpdoc spaces and short array syntax.